benchmarks/unit_benchmarks.py

```diff
@@ -0,0 +1,98 @@
+import pybamm
+import numpy as np
+
+R = 1e-5
+
+
+def time_create_expression():
+    global model
+    model = pybamm.BaseModel()
+
+    R = pybamm.Parameter("Particle radius [m]")
+    D = pybamm.Parameter("Diffusion coefficient [m2.s-1]")
+    j = pybamm.Parameter("Interfacial current density [A.m-2]")
+    F = pybamm.Parameter("Faraday constant [C.mol-1]")
+    c0 = pybamm.Parameter("Initial concentration [mol.m-3]")
+
+    c = pybamm.Variable("Concentration [mol.m-3]", domain="negative particle")
+    N = -D * pybamm.grad(c)
+    dcdt = -pybamm.div(N)
+    model.rhs = {c: dcdt}
+
+    lbc = pybamm.Scalar(0)
+    rbc = -j / F / D
+    model.boundary_conditions = {
+        c: {"left": (lbc, "Neumann"), "right": (rbc, "Neumann")}
+    }
+
+    model.initial_conditions = {c: c0}
+    model.variables = {
+        "Concentration [mol.m-3]": c,
+        "Surface concentration [mol.m-3]": pybamm.surf(c),
+        "Flux [mol.m-2.s-1]": N,
+    }
+
+
+def setup_parameterise():
+    time_create_expression()
+
+
+def time_parameterise():
+
+    global param
```

Comment:
you should be able to do this without using `global`. I have never seen a good reason to use it

Reply:
The benchmarks don't work without these global variables even after adding the setup functions. I don't know why this is the case, but when I run the benchmarks with the global variables they work